packages/express/mixin.core.js

```diff
@@ -36,42 +56,40 @@ class ExpressMixin extends Mixin {
       }, {})
     );
   }
+  configureServer(app, middlewares, mode) {
+    if (mode !== 'static') {
+      const helmet = require('helmet');
+      middlewares.initial.push(helmet());
+      middlewares.files.push(this.createStaticMiddleware());
```

Comment:
`if (mode !== 'static')` => `createStaticMiddleware` we might want to rethink about the naming here  😄